oci/runtime_oci.go

```diff
@@ -794,6 +794,9 @@ func (r *runtimeOCI) AttachContainer(c *Container, inputStream io.Reader, output
 	case err := <-receiveStdout:
 		return err
 	case err := <-stdinDone:
+		if !c.StdinOnce() && !tty {
```

Comment:
this is failing now with the upstream test.  I've opened a PR: https://github.com/cri-o/cri-o/pull/3801